### ISSUE TITLE
GD Perl module error

### ISSUE THREAD

**Issue by vanzod** [hpcugent/easybuild-easyconfigs#4394]
I am creating `GD-2.56-GCC-5.4.0-2.26-Perl-5.24.0.eb` based on the existing easyconfig `GD-2.52-ictce-5.5.0-Perl-5.18.2.eb`. However it fails with the following error:

```
"/gpfs0/software/centos6/software/Compiler/GCC/5.4.0-2.26/Perl/5.24.0/bin/perl" "-Iblib/arch" "-Iblib/lib" bdf_scripts/bdf2gdfont.PLS bdf_scripts/bdf2gdfont.pl
Can't create bdf2gdfont.pl: Permission denied at bdf_scripts/bdf2gdfont.PLS line 11.
make: *** [bdf_scripts/bdf2gdfont.pl] Error 13
```

Based on [this ticket](https://rt.cpan.org/Public/Bug/Display.html?id=99901) it seems to be a known issue of the `Makefile.PL` and they suggest to use the `Build.PL` instead. I tried the route of adding `bdf_scripts/bdf2gdfont.pl` to `MANIFEST.SKIP` but it did not work for me.

**Comment by boegel** [comment]
Very much related to #4357, @SethosII thoughts on this?

**Comment by SethosII** [comment]
@vanzod I'm also struggling to get the current version of GD to work (for Circos). I could get beyond this point by removing bdf2gdfont.pl from the archive or by making it writable. The problem is that the file isn't writable for anyone in the archive (permissions are 555).

Nevertheless afterwards I ran into problems with tests. See the [pull request](https://github.com/hpcugent/easybuild-easyconfigs/pull/4357) referenced by @boegel.

**Comment by vanzod** [comment]
@SethosII Yes, the problem is that with the generated Makefile no actual compilation is performed and no `GD.so` file is generated, raising the errors you have seen. I have manually tested by using the recommended `Build.PL` way which generated the object file but still failed through the tests. Aside from this, I have not been able to find a way to set a prefix path via `Build.PL`. For this reason I have opened this issue, hoping to get some feedback from the developers:

https://github.com/lstein/Perl-GD/issues/19

**Comment by SethosII** [comment]
@vanzod Please have a look at the pull request https://github.com/hpcugent/easybuild-easyconfigs/pull/4712.

**Comment by boegel** [comment]
Can this be closed thanks to @SethosII PR #4712?

**Comment by vanzod** [comment]
Yes, issue is solved. Closing
